src/Translation/English/translation.json

```diff
@@ -233,23 +233,22 @@
       "greetings": "Dear",
       "salutations": "Kind regards,<br>Your Portato Team"
     }
-
   },
   "navigationButton": {
     "next": "Next",
     "back": "Back",
     "confirm": "Confirm",
-    "signIn": "Sign In",
+    "signIn": "Sign in",
     "error": "Please complete the input field(s)."
   },
   "navigationMenu": {
     "myAccount": "My Account",
     "mySendRequests": "My Send Requests",
-    "signIn": "Sign In",
+    "signIn": "Sign in",
     "blog": "Blog",
-    "support": "Support / Feedback",
+    "support": "Support / feedback",
     "adminWindow": "Admin Window",
```

Comment:
For consistency's sake probably worth going with "Admin window", "My account", "My send requests", etc. as well?